backends/tfhe-cuda-backend/cuda/include/pbs/pbs_utilities.h

```diff
@@ -9,20 +9,29 @@
 template <typename Torus>
 uint64_t get_buffer_size_full_sm_programmable_bootstrap_step_one(
     uint32_t polynomial_size) {
-  return sizeof(Torus) * polynomial_size +      // accumulator_rotated
-         sizeof(double2) * polynomial_size / 2; // accumulator fft
+  size_t scalar_size = sizeof(Torus);
```

Comment:
I think we could keep just `sizeof(Torus)` in the formula, it seems easy to read to me, wdyt?

Reply:
Yes I agree.